docassemble/MassAppealsCourt/data/questions/appeals-basic-questions.yml

```diff
@@ -136,39 +136,41 @@ question: |
   Final Review With Cover Page
 subquestion: |
   
-  Below is your ${comma_and_list(al_download_titles)} document with the
-  cover page that we will deliver to ${courts[0]}.
+  Below is your document with the 
+  cover page that will be delivered to the ${courts[0]}.
```

Comment:
I'm okay with removing the titles, but we prefer [active voice](https://suffolklitlab.org/docassemble-AssemblyLine-documentation/docs/content_and_plain_language_guide#write-in-active-voice-not-passive-voice), not passive voice, right?

Reply:
Changed in new commit. Should definitely be active voice.